website/docs/r/third_party_integration.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: third_party_integration"
+sidebar_current: "docs-mongodbatlas-datasource-third-party-integration"
+description: |-
+     Provides a Third-Party Integration Settings resource.
+---
+
+# mongodbatlas_third_party_integration
+
+`mongodbatlas_third_party_integration` Provides a Third-Party Integration Settings for the given type.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+-> **NOTE:** Slack integrations now use the OAuth2 verification method and must be initially configured, or updated from a legacy integration, through the Atlas third-party service integrations page. Legacy tokens will soon no longer be supported.

```

Comment:
Can you add the link to the integrations page:
https://docs.atlas.mongodb.com/tutorial/third-party-service-integrations/

Slack integrations now use the OAuth2 verification method and must be initially configured, or updated from a legacy integration, through the [Atlas third-party service integrations page](https://docs.atlas.mongodb.com/tutorial/third-party-service-integrations/). Legacy tokens will soon no longer be supported.